index.js

```diff
@@ -136,31 +136,49 @@ client.on('messageCreate', async (message) => {
                 const m = Math.floor(s / 60);
                 const h = Math.floor(m / 60);
                 const d = Math.floor(h / 24);
-                await message.reply(`Live for ${d} day(s), ${h - (d * 24)} hour(s), ${m - (h * 60) - (d * 24 * 60)} minute(s) and ${s - (m * 60) - (h * 60 * 60) - (d * 24 * 60 * 60)} second(s).`);
+                await message.reply(`Live for ${d} day(s), ${h - (d * 24)} hour(s), ${m - (h * 60)} minute(s) and ${s - (m * 60)} second(s).`);
                 break;
               }
+              case 'help':
+                await message.reply('[[card_name]] - Will search for a card in Hero Realms, and '
+                                  + 'will provide the results in the default view.  If there are '
+                                  + 'more than one result, a dropdown will appear to where you '
+                                  + 'can pick the actual card.\n'
+                                  + '[[t:card_name]] - Will return only the text of a given '
+                                  + 'card (no image).\n'
+                                  + '[[i:card_name]] - Will only return the image of a given '
+                                  + 'card.\n'
+                                  + '[[v:card_name]] - Will show the most verbose display for a '
+                                  + 'given card.');
+                break;
               case 'pint':
                 await message.reply('I\'ll raise a glass to that!  Not that bots can actually drink a pint....');
                 break;
-              case 'help':
-                await message.reply('I\'m sorry Dave, but I can\'t allow you to do that.');
+              case 'open':
+                await message.reply('I\'m sorry, Dave.  I\'m afriad I can\'t do that.');
+                break;
+              case 'cake':
+                await message.reply('The cake is not a lie.');
```

Comment:
But it is!

Reply:
If you want to change it, that's fine :)